client/app.jsx

```diff
@@ -49,6 +58,26 @@ export default class App extends React.Component {
     if (this.state.isAuthorizing === true) {
       return null;
     }
+
+    if (window.location.hash === '#artists-page' || this.state.clickedOn === 'artists-page') {
+      return (
+        <>
+          <Artists />
+          <div id="nav-bar">
+            <div className="column-one-third">
+              <i className="fa-solid fa-house icon" onClick={this.handleClick}></i>
+            </div>
+            <div className="column-one-third">
+              <i className="fa-solid fa-music icon" onClick={this.handleClick}></i>
+            </div>
+            <div className="column-one-third">
+              <i className="fa-solid fa-user-group icon" onClick={this.handleClick}></i>
+            </div>
+          </div>
```

Comment:
Ok, i believe that this navbar can now become its own component. And instead of a `onClick`, put the `<i>` in an `<a>` with an `href="#hash-route-to-go-to"`

Reply:
Thanks for the feedback.

I went ahead and separated the "navbar" onto its own component. I also implemented hash routing rather than using onClick.

Let me know if there's anything else!